spec/factories/user_factory.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    email { Faker::Internet.email }
+    password { '0000000' }
+    password_confirmation { '0000000' }
```

Comment:
Here to be sure to have the same string than `password` call the variable directly :)

Reply:
I didn't think of doing that. Good idea.